templates/dokuwiki.nomad

```diff
@@ -25,6 +26,12 @@ job "dokuwiki" {
           php = 80
         }
       }
+      env {
+        LIQUID_CORE_URL = "${config.liquid_core_url}"
```

Comment:
We write this as `${liquid_core_url|tojson}` now. Altough I prefer your version visually.

https://github.com/liquidinvestigations/node/blob/eb956246080f4afa77bc34c758a77ba7fabd39bc/templates/hypothesis.nomad